spec/features/index_tests/user_views_posts_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature 'user views the home page', %{
+  As a user
+  I want to view a list of meetup posts
+  so I can meetup with other users
+  } do
+
+    before :each do
+      user = FactoryGirl.create(:user)
+    end
+    scenario 'unauthorized user views a list of posts' do
+      meetup_1 = FactoryGirl.create(:meetup)
+      meetup_2 = FactoryGirl.create(:meetup)
+      meetup_3 = FactoryGirl.create(:meetup)
+
+      visit meetups_path
+      expect(page).to have_content(meetup_1.title)
+      expect(page).to have_content(meetup_2.title)
+      expect(page).to have_content(meetup_3.title)
+    end
+
+    scenario 'authorized user views a list of posts' do
+
+      meetup_1 = FactoryGirl.create(:meetup)
+      meetup_2 = FactoryGirl.create(:meetup)
+      meetup_3 = FactoryGirl.create(:meetup)
+
+      visit meetups_path
+      expect(page).to have_content(meetup_1.title)
+      expect(page).to have_content(meetup_2.title)
+      expect(page).to have_content(meetup_3.title)
+    end
+
+    scenario 'user uses infinite scroll', :js => true do
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.